android-todo-list/architecture-samples-main/app/build.gradle.kts

```diff
@@ -25,14 +28,19 @@ plugins {
 android {
     compileSdk = libs.versions.compileSdk.get().toInt()
 
+    val apikeyPropertiesFile = rootProject.file("apikey.properties")
+    val apikeyProperties = Properties()
+    apikeyProperties.load(FileInputStream(apikeyPropertiesFile))
+
     defaultConfig {
         applicationId = "com.example.android.architecture.blueprints.main"
         minSdk = libs.versions.minSdk.get().toInt()
         targetSdk = libs.versions.targetSdk.get().toInt()
         versionCode = 1
         versionName = "1.0"
 
-        testInstrumentationRunner = "com.example.android.architecture.blueprints.todoapp.CustomTestRunner"
+        testInstrumentationRunner =
+            "com.example.android.architecture.blueprints.todoapp.CustomTestRunner"
```

Comment:
Can we change the package name to "com.statsig.todoapp"

Reply:
Okay I will try to change it to "com.statsig.todoapp"